Author: Michael Vogt <michael.vogt@ubuntu.com>
Description: Set max number of virtual desktops to 1
 Set max number of virtual desktops to 1. This does not affect
 viewports (and big virtual desktops) and avoids confusion when
 based on the viewport<->workspace difference between   
 compiz/metacity (LP: #118223)
Origin: vendor, ubuntu (1:0.7.0-0ubuntu3)
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/compiz/+bug/118223
--- compiz.orig/metadata/core.xml.in.in
+++ compiz/metadata/core.xml.in.in
@@ -227,7 +227,7 @@
 		    <_long>Number of virtual desktops</_long>
 		    <default>1</default>
 		    <min>1</min>
-		    <max>36</max>
+		    <max>1</max>
 		</option>
 	    </group>
 	    <group>
